src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/translation.html.twig

```diff
@@ -156,6 +156,31 @@
         {% endblock messages %}
     {% endif %}
 
+    {% if collector.globalParameters is not empty %}
+        <h2>Global parameters</h2>
+
+        <table>
+            <thead>
+            <tr>
+                <th>Locale</th>
+                <th>Message ID</th>
+                <th>Value</th>
+            </tr>
+            </thead>
+            <tbody>
+            {% for locale, values in collector.globalParameters %}
+                {% for id, value in values %}
+                    <tr>
+                        <td class="font-normal text-small nowrap">{{ locale != '*' ? "#{locale}" }}</td>
```

Comment:
```suggestion
                        <td class="font-normal text-small nowrap">{{ locale != '*' ? "{locale}" }}</td>
```

To stay consistent with existing table in translation panel
<img width="404" alt="image" src="https://github.com/user-attachments/assets/2e364e04-7cf6-4cff-aad9-77e3c21c835f" />

Reply:
I'm not sure to understand. If we remove `#`, the string interpolation won't be done and it will lead to a column with `{locale}` values.